pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -87,6 +88,10 @@
   bpftools,
   libbpf,
 
+  # vmlinux.h file containing the kernel's BTF information,
+  # used throughout systemd's BPF programs.
+  vmlinux-h ? "${callPackage ./vmlinux-btf.nix { }}/vmlinux.h",
```

Comment:
I think I would like to see this added to the passthru if only so that it's easy to build it directly or read its build log.

Reply:
Oh, it's a thin `runCommand` wrapper around the actual `vmlinux` derivation. I guess that should be in the `runCommand`'s `passthru` as well.